app/models/step_by_step_page.rb

```diff
@@ -128,6 +128,10 @@ def links_last_checked_date
     steps.map(&:links_last_checked_date).reject(&:blank?).max
   end
 
+  def links_checked_since_last_update?
+    links_checked? && links_last_checked_date > draft_updated_at
```

Comment:
Have just done some more testing locally and it looks as though scheduling updates the `draft_updated_at` value.

So you would:

- Have an inset prompt stating you should check links
- You check links, getting rid of the prompt
- You successfully schedule the step by step
- You now have another inset prompt stating you should check links

Is there another field I can use instead I wonder...? 🤔